frontend/src/Components/App/App.jsx

```diff
@@ -1,26 +1,37 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.css';
 import { Route, Routes, BrowserRouter } from 'react-router-dom';
 import Navbar from '../Navbar/Navbar';
 import Footer from '../Footer/Footer';
-import Stream from '../Pages/Stream/Stream';
-import Store from '../Pages/Store/Store';
-import AdminPanel from '../Pages/AdminPanel';
+import {
+  Stream, Store, AdminPanel, Landing,
+} from '../Pages';
+import { msToTime } from '../../Utils';
 // import PlayerData from '../PlayerData';
 
 const App = () => {
+  const streamDate = new Date('April 8, 2022 19:00:00');
   const [selectedPlayer, setSelectedPlayer] = useState('');
+  const [remainingTime, setRemainingTime] = useState(streamDate - Date.now());
+  const [countdown, setCountdown] = useState(msToTime(remainingTime));
+
+  useEffect(() => {
+    setTimeout(() => {
+      setRemainingTime(streamDate - Date.now());
+      setCountdown(msToTime(remainingTime));
```

Comment:
Why do you need two countdown variables? `setCountdown` is redundant

Reply:
because im dumb and didn't remove that